subcommittees/mentoring/procedure-discussion-session.md

```diff
@@ -0,0 +1,96 @@
+
+This document describes how instructor discussion sessions
+are organized and conducted.
+
+## Terminology
+
+-   **SCF**: Software Carpentry Foundation.
+
+-   **DC**: Data Carpentry.
+
+-   **Instructor discussion session**: an online meeting
+    where instructors share experiences from teaching and
+    obtain information while preparing to teach.
+
+-   **Facilitator**: is a member of the Mentoring Subcommittee
+    who hosts a discussion session.
+
+## Motivation
+
+1.  SCF and DC are community based projects
+    and the community needs opportunity to interact.
+
+2.  SCF and DC believe in jugyokenkyu, or "lesson study",
+    and evidence-based education so instructors need opportunity to share experiences.
+
+3. SCF and DC are continuing to increase the number of instructors
+   and develop new lessons. New instructors would like to hear from 
+   experienced teachers who have recently taught to help prepare for upcoming workshops.
+
+## Before the meeting
+
+The Mentoring Subcommittee is responsible for organizing
+the instructor discussion sessions so that
+
+-   each month has at least four discussion sessions,
+
+-   instructors are notified at least one week in advance and
+
+-   each session has at least one facilitator (preferably two) to conduct the discussion.
+
+Meeting scheduling will be coordinated via [etherpad](http://pad.software-carpentry.org/instructor-discussion).
+
+The dates of instructor discussion sessions are also listed
+in the SCF community calendar,
+https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c47g%40group.calendar.google.com.
+
+Email invitations should be sent to three groups of instructors:
+
+- instructors who have taught in the previous two weeks
+
+- instructors who are preparing to teach in the next 4-6 weeks
+
+- folks who have recently completed instructor training and need to
+  attend a discussion session to complete their checkout
+
+The contact information for the first two groups is available in [AMY](https://amy.software-carpentry.org/workshops/). 
+The chair of the mentoring subcommittee, as well as all members of the SCF Steering committee, 
+should have access to AMY. Newly trained instructors should be directed to instructor discussion sessions 
+following completion of instructor training.
+
+## During the meeting
+
+Meetings will be hosted via [etherpad](http://pad.software-carpentry.org/instructor-discussion).
+
+Each instructor discussion session session must have at least one facilitator
+from the mentoring subcommittee (although two hosts are preferred). 
+
+The **facilitator** is responsible for: 
+
+- hosting the meeting and facilitating discussion using the agenda described on the 
+[etherpad](http://pad.software-carpentry.org/instructor-discussion). 
+
+- ensuring instructors teaching in the near future have urgent questions or concerns addressed.
+
+- aggregating feedback from instructors who have recently taught.
```

Comment:
Not sure what 'aggregating' exactly means in this context. Is this just a summary or are you envisioning something more specific?